src/external-interface.ts

```diff
@@ -1,62 +1,46 @@
-interface IBls {
-  Keypair: {
-    generate(): Keypair;
-  };
+export interface IBls {
   PrivateKey: {
-    fromBytes(bytes: Uint8Array): PrivateKey;
-    fromHexString(value: string): PrivateKey;
-    fromInt(num: number): PrivateKey;
-    random(): PrivateKey;
+    fromKeygen(ikm?: Uint8Array): IPrivateKey;
+    fromBytes(bytes: Uint8Array): IPrivateKey;
+    fromHex(hex: string): IPrivateKey;
```

Comment:
- fromRandom
- fromInt

Reply:
fromInt does not exist in BLST

Reply:
it's easy to implement in js, we use that extensively in tests

Reply:
`fromRandom` excludes the idea of generating from input key material, which is what fromKeygen does. Herumi does not support that, but BLST does

Reply:
How do you plan on implementing that on herumi side then?